client/src/main/java/com/connectifier/xeroclient/XeroClient.java

```diff
@@ -336,9 +335,9 @@ public Invoice getInvoice(String id) {
     return post("Invoices", objFactory.createInvoice(invoice)).getInvoices();
   }
 
-  public List<Receipt> createReceipt(Receipt receipt) {
-    return put("Receipts", objFactory.createReceipt(receipt)).getReceipts();
-  }
+//  public List<Receipt> createReceipt(Receipt receipt) {
```

Comment:
I see. It's described in https://github.com/benmccann/xero-java-client/pull/9

I can't merge https://github.com/benmccann/xero-java-client/pull/9 currently because it requires a rebase, but I like the solution there better